Auth0/_ObjectiveAuthenticationAPI.swift

```diff
@@ -76,11 +77,13 @@ public class _ObjectiveAuthenticationAPI: NSObject {
                 }
             }
     }
-    
+
+#if canImport(UIKit)
     @objc(resumeAuthWithURL:options:)
     public static func resume(_ url: URL, options: [A0URLOptionsKey: Any]) -> Bool {
         return resumeAuth(url, options: options);
```

Comment:
```suggestion
        return resumeAuth(url, options: options)
```
Swiftlint throws a warning about the semicolon.